src/__main__.py

```diff
@@ -0,0 +1,7 @@
+import pyglet
+
+from game import update
```

Comment:
```suggestion
from .game import update
```
Testing locally, I needed to add a `.` to run the game with `py -m src`